server/website/website/models.py

```diff
@@ -251,6 +251,27 @@ def get_knobs_for_session(session):
         knob_dicts = [knob for knob in knob_dicts if knob["tunable"]]
         return knob_dicts
 
+    @staticmethod
+    def get_knob_tunability(session):
+        # Returns a dict of the knob
+        knobs = KnobCatalog.objects.filter(dbms=session.dbms)
+        knob_dicts = list(knobs.values())
+        session_knob_dict = {}
+        for i, _ in enumerate(knob_dicts):
+            if SessionKnob.objects.filter(session=session, knob=knobs[i]).exists():
+                new_knob = SessionKnob.objects.filter(session=session, knob=knobs[i])[0]
+                session_knob_dict[new_knob.name] = new_knob.tunable
+        return session_knob_dict
+
+    @staticmethod
+    def set_knob_tunability(session, knob_dicts):
```

Comment:
Similar to my comment above, we also need to be able to set the min/max values for each tunable session knob.